lib/fabrik/DirectorManager.js

```diff
@@ -211,6 +211,31 @@ class DirectorManager extends BaseManager {
                 previous_manifest: manifest
               }, opts.context);
               return;
+            })
+            .then(() => {
+              let updateActionContext = _.cloneDeep(actionContext);
+              _.chain(updateActionContext)
```

Comment:
using chain but using a single operation? why not use a single set operation directly?

Reply:
also why are we cloning this?